include/ttmlir/Conversion/Passes.td

```diff
@@ -7,6 +7,13 @@
 
 include "mlir/Pass/PassBase.td"
 
+def ConvertStableHLOToTTIR : Pass<"convert-stablehlo-to-ttir", "::mlir::ModuleOp"> {
```

Comment:
Should be surrounded with `#ifdef TTMLIR_ENABLE_STABLEHLO`

Reply:
Ok that's the problematic case, the tablegen doesn't recognize #ifdef TTMLIR_ENABLE_STABLEHLO, so it is always disabled after adding it. I have to find another way to transfer this option to tablegen.